src/types/profile.js

```diff
@@ -375,6 +375,12 @@ export type ProfileMeta = {|
   logicalCPUs?: number,
   // A boolean flag for whether or not the network URLs were stripped from the profile.
   networkURLsRemoved?: boolean,
+  // A boolean flag indicating whether we symbolicated this profile. If this is
+  // false we'll start a symbolication process when the profile is loaded.
+  // `null` happens for older profiles, it stands for an "unknown" state. For
+  // now we don't do much with it but we may want to propose a manual
+  // symbolication in the future.
+  symbolicated: boolean | null,
```

Comment:
Optional: I would prefer something along the lines of:

`symbolicatationStatus: "SYMBOLICATED" | "UNSYMBOLICATED" | "UNKNOWN"`

as it makes the state explicit. The `null` value to me doesn't imply much meaning here to me. Or maybe, do:

`symbolicated?: boolean`

Which would at least imply that it's not known, as the property isn't there. I believe this would also avoid a version bump.

Reply:
Mostly I forced `null` so that Flow gave me all places that needed changes. (obviously I missed some). I wanted to explicitely handle this case.

I'll sleep on your suggestions and decide tomorrow :)

Reply:
I decided to go with the simpler `symbolicated?: boolean` and removed the profile upgrade.